tests/security/tpm2/tpm2_env_setup.pm

```diff
@@ -62,10 +63,14 @@ EOF
     $self->select_serial_terminal;
 
     # Start the emulator
-    assert_script_run "su - tss -c '/usr/lib/ibmtss/tpm_server&'";
+    if (is_sle) {
+        assert_script_run "su - tss -c '/usr/lib/ibmtss/tpm_server&'";
+    } else {
+        assert_script_run "su - tss -c '/usr/libexec/ibmtss/tpm_server&'";
+    }
 
```

Comment:
nit: I'd define server path as variable:

```
my $server = '/usr/libexec/ibmtss/tpm_server';
$server = '/usr/lib/ibmtss/tpm_server' if (is_sle);
assert_script_run "su - tss -c '$server &'";
```
Or even detect it automatically:
```
my $server = script_output('f=/usr/libexec/ibmtss/tpm_server; [ ! $f ] && f=/usr/lib/ibmtss/tpm_server; echo $f');
die 'missing tpm_server path\n' if ($server eq '');
assert_script_run "su - tss -c '$server &'";
```

Reply:
Thanks for the suggestion!

Reply:
or even more generic (it'd work on any distro changes in the future):

```
my $server = script_output('ls /usr/*/ibmtss/tpm_server |tail -1');
die 'missing tpm_server path\n' if ($server eq '');
assert_script_run "su - tss -c '$server &'";
```